lms/static/js/verify_student/views/image_input_view.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Allow users to upload an image using a file input.
+ *
+ * This uses HTML Media Capture so that iOS will
+ * allow users to use their camera instead of choosing
+ * a file.
+ */
+
+ var edx = edx || {};
+
+ (function( $, _, Backbone, gettext ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.ImageInputView = Backbone.View.extend({
+
+        template: "#image_input-tpl",
+
+        initialize: function( obj ) {
+            this.$submitButton = obj.submitButton || "";
+            this.modelAttribute = obj.modelAttribute || "";
+            this.errorModel = obj.errorModel || null;
+        },
+
+        render: function() {
+            var renderedHtml = _.template( $( this.template ).html(), {} );
+            $( this.el ).html( renderedHtml );
+
+            // Set the submit button to disabled by default
+            this.setSubmitButtonEnabled( false );
+
+            this.$input = $( 'input.image-upload' );
+            this.$preview = $( 'img.preview' );
+            this.$input.on('change', _.bind( this.handleInputChange, this ) );
+
+            // Initially hide the preview
+            this.displayImage( false );
+
+            return this;
+        },
+
+        handleInputChange: function( event ) {
+            var files = event.target.files,
+                reader = new FileReader();
+            if ( files[0] && files[0].type.match( 'image.[png|jpg|jpeg]' ) ) {
+                reader.onload = _.bind( this.handleImageUpload, this );
+                reader.onerror = _.bind( this.handleUploadError, this );
+                reader.readAsDataURL( files[0] );
+            }
+            else if ( files.length === 0 ) {
```

Comment:
nitpick: JavaScript formatting should have the closing if bracket on the same line as the else if